packages/server/modules/activitystream/services/branchActivity.ts

```diff
@@ -38,8 +37,7 @@ export const addBranchCreatedActivityFactory =
         info: { branch },
         message: `Branch created: ${branch.name} (${branch.id})`
       }),
-      // @deprecated
```

Comment:
not really, Alessandro added it to avoid having to type these old subscriptions but i've done it here anyway. besides, there's a bunch of other subscriptions that are also "deprecated" and don't have this tag